cmd/lachesis/config_migrations.go

```diff
@@ -59,40 +59,67 @@ func (d *ConfigData) GetTable() *ast.Table {
 }
 
 func (d *ConfigData) AddSection(name, after string) error {
+	if name == "" {
+		return nil
+	}
+
 	_, err := d.findSection(name)
 	if err == nil {
 		// If exists - return error
 		return errors.New("section already exists: "+name)
 	}
 
+	path := strings.Split(name, ".")
+
 	afterSection, err := d.findSection(after)
 	if err != nil {
 		return err
 	}
 
-	newSection := &ast.Table{
-		Position: ast.Position{
-			Begin: afterSection.End() + 1,
-			End:   0,
-		},
-		Line:     afterSection.Line + afterSection.End() - afterSection.Pos(),
-		Name:     name,
-		Fields:   make(map[string]interface{}),
-		Type:     ast.TableTypeNormal,
-	}
+	pathStr := ""
```

Comment:
it seems (line 72 and 79,81,82) is equivalent to:
pathStr := "/" + strings.ReplaceAll(name, ".", "/")